sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveUDFSuite.scala

```diff
@@ -552,6 +552,14 @@ class HiveUDFSuite extends QueryTest with TestHiveSingleton with SQLTestUtils {
     }
   }
 
+  test("Invoke a persist hive function with call_function") {
+    val testData = spark.range(5).repartition(1)
+    withUserDefinedFunction("custom_func" -> false) {
+      sql(s"CREATE FUNCTION custom_func AS '${classOf[GenericUDAFAverage].getName}'")
+      checkAnswer(testData.select(call_function("custom_func", $"id")), Row(2.0))
```

Comment:
can we also test calling the function with the qualified name? `spark_catalog.default.custom_func`